verilog/CST/expression.cc

```diff
@@ -98,6 +98,25 @@ const verible::Symbol* GetConditionExpressionFalseCase(
   return GetSubtreeAsSymbol(condition_expr, NodeEnum::kConditionExpression, 4);
 }
 
+const verible::TokenInfo* GetUnaryPrefixOperator(
+    const verible::Symbol& symbol) {
+  const SyntaxTreeNode* node = &verible::SymbolCastToNode(symbol);
```

Comment:
this will crash if symbol is not a node.

Reply:
Check added.